src/main/java/core/mvc/JsonView.java

```diff
@@ -1,12 +1,56 @@
 package core.mvc;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.Map;
 
 public class JsonView implements View {
+
+    private int statusCode;
+    private String url;
+
+    public JsonView(int statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public JsonView(int statusCode, String url) {
+        this.statusCode = statusCode;
+        this.url = url;
+    }
+
     @Override
     public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        final String body = getBody(model);
+        final int contentLength = body.getBytes().length;
+
+        response.setStatus(statusCode);
+        if (url != null) {
+            response.setHeader("Location", url);
+        }
+        response.setContentType("application/json;charset=UTF-8");
+        response.setContentLength(contentLength);
+        response.getWriter().write(body);
+    }
+
+    private String getBody(Map<String, ?> model) throws JsonProcessingException {
+        final ObjectMapper objectMapper = new ObjectMapper();
+
+        if (model.isEmpty()) {
+            return "";
```

Comment:
직접 값을 넣기 보다는 정의되어 있는 상수를 활용하는 것이 가독성을 증가시킨다고 생각해요 :)
```suggestion
            return StringUtils.EMPTY;
```